bionic/flow.py

```diff
@@ -959,6 +959,23 @@ def get(self, name, collection=None, fmt=None, mode=object):
         else:
             raise ValueError("Unrecognized collection type %r" % collection)
 
+    def get_docstring_for_entity(self, name):
```

Comment:
We might as well match the naming convention of `Flow.entity_protocol`, which does something similar.